package.json

```diff
@@ -63,6 +64,7 @@
     "micromark-util-combine-extensions": "^1.0.0",
     "moment-parseformat": "^4.0.0",
     "quick-lru": "^6.1.1",
-    "ts-dedent": "^2.2.0"
+    "ts-dedent": "^2.2.0",
+    "unist-util-visit": "^4.1.2"
```

Comment:
This should already be included by another package. I say that because it has run just fine up until now. I do not mind adding it, but I am not sure why we would need to add it now.

Reply:
It could be that these changes were somehow included globally for me and that is why they work for me. But they seem to work in `main.js` for the plugin. Could you let us know what editor you used @qazxcdswe123 that needed these npm packages to work?

Typically I use VSCode or NeoVim. So it could be that one of them already accounted for the discrepancy in dependencies.

Reply:
This is what I get after removing this two dependencies, the build/test can't be run. 

I use neovim and the lsp also compliant about the import.

<img width="1440" alt="image" src="https://github.com/platers/obsidian-linter/assets/29861494/1e3db08b-d3d8-4285-ab50-1bee6c63159c">

<img width="1440" alt="image" src="https://github.com/platers/obsidian-linter/assets/29861494/c67093be-d1b9-41fe-9b2e-f78f55431770">

Reply:
Gotcha. It should be fine to merge this then. My node modules locally must have kept those values somehow.

Reply:
BTW, mayble a different version of this dependency is needed since the UI somehow broke for me 😢